chapter1/item2.md

```diff
@@ -2,4 +2,56 @@
 
 *This item covers customization using X resource files and tools*
 
-TODO
+---
+
+X resources provide a simple yet powerful way to customize your X session. With files such as `.Xresources`, you can easily assign colors and fonts globally or for specific applications. 
+
+_NB_: If you have a `.Xdefaults` file as opposed to a `.Xresources`, it may be a good idea to change it. `.Xdefaults` is reloaded everytime an X program starts, therefore less eddicient.
+
+Most often, it is used to set colors globally and then application specific options, typically for a terminal. `urxvt` is a common item that has its options set in `.Xresources`. The syntax is quite simple,
+makeing it easy to read *and* write. The syntax typically follows `appname*resource*subresource...*subresource: value`. If you want a setting to apply to all applications that use the X Resources, 
+don't include an `appname` and prefix it only with *. This is typically a good idea when setting colors, since you likely want a uniform look throughout your system. For other options, such as fonts, it
+might be a better idea to set the on a per program basis. That way, you could use a different font for you bar and terminal.
+
+Classes are also a feature of X resources. They allow you to set multiple options at once. Classes are always capitalized, such as `xclock*Foreground: #ffffff` will set `xclock*foreground`, `xclock*hands`, and 
+`xclock*highlight` to the hex color `#ffffff`. I wont go into a lot of detail on these, as they are different
+for each program. You can find list [here](https://stuff.mit.edu/afs/sipb/project/doc/ixresources/xres.html#extraresources).
+
+_NB_: If you are familiar with C preprocessor constructs, you can make use of `#ifdef`, `#else`, and `#endif` here. 
+
+---
+
+Now, you amy think "Well, I'll set my colors in `.Xresources` and be good to go!". Not quite. We have to load these options somehow - turns out, there is already a tool to do so.
```

Comment:
That wasn't the kind of explanation I mentioned. I meant something like "go through all the options and make the readers feel they are also doing it". Yours is no different than a manual page.